src/app/app.component.ts

```diff
@@ -11,15 +11,19 @@ export class AppComponent {
   fhirPreview: string;
   questionnaire = 'bmi';
   fhir = fhir;
-  linkId = context;
+  phq9ScoreLinkId = phq9ScoreLinkId;
```

Comment:
This is not only specific to the phq9. Also removing linkId makes the e2e tests fail.

Reply:
Fixed

Reply:
Also what does the variable phq9ScoreLinkId do? (Not the one in mock-data)